update loading CSS (#3846)
webpack/webpack.js.org: src/content/guides/asset-management.md

+   },
  };
```

> Module loaders can be chained. Each loader in the chain applies transformations to the processed resource. A chain is executed in reverse order. The first loader passes its result (resource with applied transformations) to the next one, and so forth. Finally, webpack expects JavaScript to be returned by the last loader in the chain. 

Reviewer: ```suggestion

Module loaders can be chained. Each loader in the chain applies transformations to the processed resource. A chain is executed in reverse order. The first loader passes its result (resource with applied transformations) to the next one, and so forth. Finally, webpack expects JavaScript to be returned by the last loader in the chain. 
```

Author: I made the changes.

> The above order or loaders should be maintained with 'style-loader' coming first and then, 'css-loader'. If this convention is not followed, webpack is likely to throw errors.

Reviewer: ```suggestion
The above order of loaders should be maintained: 'style-loader' comes first and followed by 'css-loader'. If this convention is not followed, webpack is likely to throw errors.
```

Author: I made the changes required. I actually could not reference this branch and hence, I created another pull request. I hope that's okay.
